### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by ankushaggarwal**
Made some updates to the documentation as well as the README file. Also updated the pytorch version in the requirements to remove security vulnerability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,25 @@
 
 # pyMechT
 
-pyMechT is a <u>Py</u>thon package for simulating the <u>mech</u>anical response of soft biological <u>t</u>issues. The focus is on flexibility of defining models (referred to as *MatModels*). The ethos of pyMechT is to create simplified virtual experimental setups. That is, a *sample* is created of which there are the following options:
-* Uniaxial extension;
-* Biaxial extension; and
-* Inflation-extension.
+pyMechT is a <u>Py</u>thon package for simulating the <u>mech</u>anical response of soft biological <u>t</u>issues. The ethos of pyMechT is to create simplified virtual experimental setups, rather than finite element analysis. Thus, varying parameters and running simulations is much faster, making it feasible to perform Bayesian inference and Markov Chain Monte Carlo analysis.
```
Comment (r-williams-2) [inline README.md:5]: analyses?

File: pymecht/__init__.py [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
+    * :class:`RandomParameters` and :class:`MCMC`: Bayesian inference by running Monte Carlo (MC) and Markov chain Monte Carlo (MCMC) simulations
+
+.. image:: drawing-1.svg 
+  :width: 300
+  :alt: Structure of pyMechT
+  :align: center
+
+This package is developed and maintained by the `Computational Biomechanics Research Group <https://userweb.eng.gla.ac.uk/ankush.aggarwal/>`_ at the University of Glasgow and is hosted at `github <https://github.com/ankushaggarwal/pymecht/>`_.
+
+The ethos of pyMechT is to create simplified virtual experimental setups, rather than finite element analysis. Thus, varying parameters and running simulations is much faster, making it feasible to perform Bayesian inference and Markov Chain Monte Carlo analysis.
```
Comment (r-williams-2) [inline pymecht/__init__.py:25]: analyses again?

**Review by r-williams-2 — APPROVED** [review]
Just a minor point about changing analysis to analyses (regarding MC and MCMC analyses). As such, I didn't select to request changes before merging since they are so minor.

**Comment by ankushaggarwal** [comment]
Thanks  Ross. I made these adjustments and also added a video to the documentation and README. Merging it now.

**Pull request merged**